### ISSUE TITLE
Bathtub classification 100%

### ISSUE THREAD

**Issue by saha223311** [GrumpyZhou/pytorch-dgcnn#1]
Hello, when I use a pretrained model (50 epoch), the classifier recognizes only the bathtub!
If this is my mistake, where can it be? thanks!

Namespace(K=10, batch=16, cat='All', ckpt='ckpt_50_0.909.pth', data_dir='data/', dataset='ModelNet10', epoch=250, gpu=0, lr=0.001, lrd_factor=0.5, lrd_step=20, network='DGCNNCls', odir='outputs', seed=7, training=False, val=5, visenv=None, vishost='localhost', visport=9333, viswin=None, weight_decay=0.0, worker=6)
Testing DGCNNCls with ckpt ckpt_50_0.909.pth
Build up DGCNNCls model, num_classes 10, K 10
Total: 5.51% Per class avg: 10.00
 Per class: ['bathtub 100.0% ', 'bed 0.0% ', 'chair 0.0% ', 'desk 0.0% ', 'dresser 0.0% ', 'monitor 0.0% ', 'night_stand 0.0% ', 'sofa 0.0% ', 'table 0.0% ', 'toilet 0.0% ']
Accuracy: 5.5066 time 51.6654s

**Comment by GrumpyZhou** [comment]
Are you sure that you really load the model? The results look like the model is not properly loaded ( in this case the code will just use random initialisation). So you can easily check whether the script loads your model. If it is, you should see lines like:

```
Testing DGCNNCls with ckpt  xxxxx(your model)
Build up DGCNNCls model, num_classes 10, K 10
Load all model parameters from state dict   # Especially this line

```

**Comment by saha223311** [comment]
Thanks a lot! Yeah, it's my stupid mistake. Plus star from me :)
